### PR TITLE
Clean and update README.md

### LINKED ISSUE #39: Update and clean up README
The current README.md can probably be cleaned up a little bit. For example, the inputs to the CLI commands should be better documented:

`deposit [amount] [key]`

`submit_block [key]`

etc etc

### PR DESCRIPTION
**Pull request by kfichter**
Closes #39 

(Almost) complete rewrite of `README.md` with more complete CLI documentation.

Comments, edits, reviews welcome.

### CODE REVIEW

**Comment by boolafish** [comment]
Tried install with this Readme.
Got stucked a little bit at:
1. leveldb
2. solidity installation

I think `leveldb` should be added to pre-installation dependancy as well. 
And it would be nice to have some solidity installation guide for mac as well (like the one in another [PR](https://github.com/omisego/plasma-mvp/pull/41)). The link for Solidity 0.4.18 doesn't have mac download : (

**Comment by kfichter** [comment]
@boolafish Thanks for the feedback, I went ahead and updated the README to include better install notes for `leveldb` and `solidity`.

File: README.md [hunk 1]
```diff
@@ -111,7 +111,7 @@ The fastest way to start playing with our Plasma MVP is by starting up `ganache-
 ```bash
 $ ganache-cli -m=plasma_mvp # Start ganache-cli
 $ make root-chain           # Deploy the root chain contract
-$ make child-chain &        # Run our child chain and server in the background
+$ make child-chain          # Run our child chain and server in the background
```
Comment (boolafish) [inline README.md]: the comment should change as well if making this change : )

**Comment by kfichter** [comment]
Whoops, you are correct! Thanks.

File: README.md [hunk 2]
```diff
@@ -1,35 +1,252 @@
...
+withdraw <blknum> <tx_pos> <utxo_pos> <key1> [<key2>]
+```
+
+#### Example
+
+```
+withdraw 1 0 0 3bb369fecdc16b93b99514d8ed9c2e87c5824cf4a6a98d2e8e91b7dd0c063304
+```
+
+### `send_tx`
```
Comment (nvonpentz) [inline README.md]: I think you mean `sync` here

**Comment by kfichter** [comment]
Thanks! Fixed.

**Comment by DavidKnott** [comment]
@kfichter Read it over, and it looks good!  Merging.

**Pull request merged**